src/webextensionadapter.cpp

```diff
@@ -122,6 +137,10 @@ QJsonObject WebExtensionAdapter::serializeStatus() {
 
   {
     Controller::State state = vpn->controller()->state();
+    if (state == Controller::StateOnPartial) {
+      state = Controller::StateOn;  // Old extensions like MAC dont know and
+                                    // dont need to know about this state.

```

Comment:
We can always update MAC to handle this if we for some reason need to

Reply:
True but also mac should not care about our internal states 😅

Reply:
Let's do that, we probably should start thinking about versioning the api too.